common/lib/capa/capa/responsetypes.py

```diff
@@ -61,6 +60,11 @@
     is_list_of_files
 )
 
+if six.PY2:
+    import random
+else:
+    import random2 as random
```

Comment:
`random2` works in both Python-2 and Python-3.  So no need for this `if` statement.